lms/templates/sysadmin_dashboard.html

```diff
@@ -59,6 +62,7 @@ <h2 class="instructor-nav">
       ## Translators: refers to http://git-scm.com/docs/git-log
       <a href="${reverse('gitlogs')}">${_('Git Logs')}</a>
       <a href="${reverse('sysadmin_task_queue')}" class="${modeflag.get('task_queue')}">${_('Task Queue')}</a>
+      <a href="${reverse('sysadmin_mgmt_commands')}" class="${modeflag.get('mgmt_commands')}">${_('MGMT Commands')}</a>
```

Comment:
the class here does;t make sense to me: doesn't `modeflag.get('mgmt_commands')` a boolean? If so, `class=true` makes 0 sense...